### PR TITLE
Support flavor

### PR DESCRIPTION
**Pull request by chemaper**
#### Reviewers

@flopezag
#### Description

Support environment variable FLAVOR to change the flavor to use.

Default flavor is m1.small. This is usually the right value to use.
#### Testing

Running the script specifying a flavor andd without specifying it.

### CODE REVIEW

File: scripts/support/create_ge_images/create_template.sh [hunk 1]
```diff
@@ -35,7 +35,7 @@ export TESTSCRIPT=test.sh
 export UPLOAD_FILE=data.tgz
 export IMAGES_DIR=~/create_ge_images/
 export IP=$(nova floating-ip-list | awk '/^\|[ ]+[0-9]+/ { print $2 }')
-
+export FLAVOR=${FLAVOR:-m1.small}
```
Comment (flopezag) [inline scripts/support/create_ge_images/create_template.sh:38]: What about if there is no m1.small

Comment (chemaper) [inline scripts/support/create_ge_images/create_template.sh:38]: This is a default value. User can change it if needed, using this change. Or can change the script. This is a shell script and the constants are at the beginning of the file.

Anyway, m1.small is in any default installation. People does not change this. We use this value directly in other scripts even without a constant.

Comment (flopezag) [inline scripts/support/create_ge_images/create_template.sh:38]: NTC

**Comment by flopezag** [comment]
LGTM

**Pull request merged**